db/migrate/20200107161405_add_featured_rank_to_user_badges.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+class AddFeaturedRankToUserBadges < ActiveRecord::Migration[6.0]
+  def change
+    add_column :user_badges, :featured_rank, :integer, null: true
+
+    execute <<~SQL
```

Comment:
will this lock up the table in a transaction? do we need to be careful here and run it outside of a transaction?

Reply:
During the transaction, it holds `RowExclusiveLock`s over the rows being updated. That means existing rows in the UserBadge table cannot be updated by any other process during this time. I think new rows could still be added, since no table-level locks area being held.

Reply:
5 seconds on a large db sounds fine to me. 5 minutes and stuff can get a bit tricky.